cms/envs/common.py

```diff
@@ -1264,7 +1264,7 @@
 
 ############## Settings for Studio Context Sensitive Help ##############
 
-HELP_TOKENS_INI_FILE = REPO_ROOT / "docs" / "cms_config.ini"
+HELP_TOKENS_INI_FILE = REPO_ROOT / "cms" / "envs" / "help_tokens.ini"
```

Comment:
@edx/devops I'm building a sandbox to test the help links now.  Can you confirm that there is nothing else you need to do regarding the help link config file move for Production?  Thank you.